requirements.txt

```diff
@@ -1,2 +1 @@
 scriptworker
-aiohttp==3.2.1
```

Comment:
This has already amended [here](https://github.com/mozilla-releng/bouncerscript/pull/25/files#diff-b4ef698db8ca845e5845c4618278f29a) but if you're faster than me in merging this, I'll rebase from master.